cli/build.gradle

```
@@ -1,15 +1,21 @@
 plugins {
-    id 'bio.terra.pfb.java-common-conventions'
+    id 'bio.terra.pfb.java-library-conventions'
+    id 'maven-publish'
+    id 'com.jfrog.artifactory' version '5.0.3'
+    id 'com.gorylenko.gradle-git-properties' version '2.3.1'
 }
 
 version 'unspecified'
```

This should use `gradle.ext.releaseVersion` from `settings.gradle`:
```suggestion
version = gradle.releaseVersion
```